cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/LocalStreamService.java

```diff
@@ -200,6 +201,9 @@ private void publishNotification(long absoluteSize) {
       try {
         notificationService.publish(streamFeed, new StreamSizeNotification(System.currentTimeMillis(), absoluteSize))
           .get();
+      } catch (ServiceUnavailableException e) {
```

Comment:
The notification system actually don't care about not able to publish a feed (which is part of the design) since it requires feed consumer to resort to polling mechanism if no feed arrives in certain time bound.

I think it's ok to just have a `catch (Throwable t)` and log it as debug.